packages/@lwc/integration-karma/test/mixed-shadow-mode/retargeting/index.spec.js

```diff
@@ -0,0 +1,31 @@
+import { createElement } from 'lwc';
+import SyntheticParent from 'x/syntheticParent';
+
+function test(name, selector) {
```

Comment:
```suggestion
function testSelector(name, selector) {
```
`test` is a jest global (alias for `it`); shadowing that name could cause confusion.